### ISSUE TITLE
Consider randomizing the resolution passed to xvfb

### ISSUE THREAD

**Issue by pyoor** [MozillaSecurity/ffpuppet#7]
It might be worth considering setting a random resolution to xvfb as this might affect layout.  Similar concept to the TTF Fuzzer outlined here:

https://media.blackhat.com/bh-eu-12/Lee/bh-eu-12-Lee-GDI_Font_Fuzzing-WP.pdf

**Comment by revskills** [comment]
Agree, eg: https://www.mozilla.org/en-US/security/advisories/mfsa2015-107/
